requirements.txt

```diff
@@ -1,2 +1,2 @@
 PyOpenSSL
-pulpcore>=3.3,<3.11
+pulpcore>=3.3,<3.13
```

Comment:
If we had to make a required plugin change to release in December, shouldn't this floor version be higher? I'm not actually sure, so I'm asking.